src/kata5.reachDestination.js

```diff
@@ -1,3 +1,11 @@
-const reachDestination = (distance, speed) => {};
+/* eslint-disable prettier/prettier */
+const reachDestination = (distance, speed) => {
+    function roundHalf(num) {
+        return Math.round(num*2)/2;
+    }
+    const approxEta = (distance/speed);
+    return `I should be there in ${roundHalf(approxEta)} hours.`
+    
```

Comment:
This assumes a few things, such as units for distance and speed (a comment would have been useful here to remove the guess work). Ill assume distance is in miles and speed is in mph.
The logic here is a bit fuzzy, and doesnt return a very accurate result. You could get a more accurate to the minute result like this:

```suggestion
  const time = distance / speed;
  const hours = Math.floor(time);
  return `I should be there in ${hours} hours + ${Math.floor(60 * (time - hours))} minutes`;
```